packages/transaction-controller/src/utils/balance-changes.ts

```diff
@@ -162,6 +164,7 @@ export async function getBalanceChanges(
 
     return {
       tokenBalanceChanges: [],
+      gasUsed: undefined,
```

Comment:
Can we return this as a separate property and put the others into `simulationData` so it's not confused as part of the `SimulationData`?